src/components/MenuPanel/MenuPanel.js

```diff
@@ -141,7 +141,13 @@ class MenuPanel extends React.PureComponent {
     const { animate, scrollVisible, systemAppsOpened } = this.state
 
     const appGroups = this.getRenderableAppGroups(appInstanceGroups)
-    const menuApps = [APP_HOME, appGroups]
+
+    const hasHomeApp = appGroups.some(app => {
+      return app.isHomeApp
+    })
+
+    const menuApps = !hasHomeApp ? [APP_HOME, appGroups] : appGroups
```

Comment:
better for clarity to use positive conditions instead of negative(aka swap the terms)

Reply:
Yes, it will be read better that way.